### PR TITLE
Fix links and typos

### PR DESCRIPTION
**Pull request by elpiel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -537,10 +537,10 @@ You can also stake through Binance and Huobi.
 
 ### Basic visual representation
 
-![Architecture](./graphs/architecture-pretty.png)
+![Architecture](https://github.com/AdExNetwork/adex-protocol/blob/master/graphs/architecture-pretty.png)
```
Comment (rori4) [inline README.md:540]: This will not link the diagram properly

File: README.md [hunk 2]
```diff
@@ -537,10 +537,10 @@ You can also stake through Binance and Huobi.
...
 
 * The box-shaped platform and AdView are client-side software
-* Round-shaped items represent parts of the AdEx peer-to-peer network (in practice, [many validators and markets may exist](/graphs/real-world.svg))
+* Round-shaped items represent parts of the AdEx peer-to-peer network (in practice, [many validators and markets may exist](https://github.com/AdExNetwork/adex-protocol/blob/master/graphs/real-world.svg))
```
Comment (rori4) [inline README.md:543]: same here. Either use https://raw.githubusercontent.com/AdExNetwork/adex-protocol/master/graphs/real-world.svg
or leave it as it is

**Review by rori4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (elpiel) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (elpiel) [reply README.md]: Fixed

**Review by rori4 — APPROVED** [review]

**Pull request merged**